user_guide/src/howcani/data/timestamps.md

```diff
@@ -1,14 +1,14 @@
 # Timestamp parsing
 
-`Polars` offers 3 time datatypes:
+`Polars` offers `3` time datatypes:
```

Comment:
We can actually update that. We now have 4. Also `pl.Duration` which is semantically the same to python `timedelta`.